packages/backend/src/submissions/dto/create-submission.dto.ts

```diff
@@ -1 +1,4 @@
-export class CreateSubmissionDto {}
+import { PartialType } from '@nestjs/mapped-types';
+import { SubmissionEntitiy } from '../entities/submission.entity';
+
+export class CreateSubmissionDto extends PartialType(SubmissionEntitiy) {}
```

Comment:
This works, but making everything optional isn't the best solution.
For example we won't need the id, check the link dtos for help.